.github/workflows/rust.yml

```diff
@@ -36,10 +36,16 @@ jobs:
       - name: build & test
         run: |
           cargo check --release
-          cargo test --release
+          # This is covered by the next step,
+          # commented out to make the job faster
+          #
+          # cargo test --release
 
       - name: build & test ocaml types
         run: |
+          # Cannot use `--all-features` here
+          # Becauses `3`, `3w` and `5w` features under `oracle` crate are orthogonal,
+          # Should this be changed?
```

Comment:
if you find a way to fix this it'd be nice!

Reply:
I think it's only because the tests are orthogonal, not the features, I can make a fix in a separate PR if u like.

Reply:
that'd be awesome

Reply:
Fixed by https://github.com/o1-labs/proof-systems/pull/329

Reply:
@mimoo - We still need `3w` (i.e. `PlonkSpongeConstantsBasic` config) in addition to `5w,` since mainnet transactions payment and delegation use this configuration.